model_extension/python scripts/temp_screen.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 11 13:54:59 2024
+
+
+
+@author: camila
+"""
+
+import numpy as np
+import math
+from matplotlib import pyplot as plt 
+
+
+#lb = 0.714 #Bjerrum length, in nm
+
+e = 1.60217663*(pow(10, -19)) #C, ectron charge
+e0 = 8.854*(pow(10, -12)) #F/m, vacuum permittivity
+kb = 1.380649*(pow(10,-23)) #J/K, boltzman cte
+
+###########################
+#
+# TEMPERATURE DEPENDENCE
+
+T0 = 298
+T1=278
+T2=288
+T3=323
+T4=348
+
+
+Tk = T1 #K, temperature
+Tc = Tk - 273.15 # temp in C
+
+T = np.linspace(5, 50, 50)
+#er = 87.74 - 0.40008*Tc + (9.398e-4)*(Tc**2) - (1.410e-6)*(Tc**3)
+
+er = 87.74 - 0.40008*T + (9.398e-4)*(T**2) - (1.410e-6)*(T**3)
```

Comment:
@camilafaccini see comments above in er correction second term and not using the 0.003 error correction